From e4c31c2eb6d03edcb093f7c56c3eeb952b63e06d Mon Sep 17 00:00:00 2001
From: Ji Luo <ji.luo@nxp.com>
Date: Thu, 30 May 2019 10:23:10 +0800
Subject: [PATCH 24/31] MA-14898 Fix build warnings
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Include correct header files to avoid build warning:
  common/image-android.c:151:68: warning: implicit declaration
  of function ‘mmc_get_env_dev’ [-Wimplicit-function-declaration]
  " androidboot.boot_device_root=mmcblk%d", mmc_map_to_kernel_blk(mmc_get_env_dev()));

Change-Id: I932a178500e7397f72d717efae0dd2d954a909d8
Signed-off-by: Ji Luo <ji.luo@nxp.com>
---
 common/image-android.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/image-android.c b/common/image-android.c
index 20b4672..6d54d42 100644
--- a/common/image-android.c
+++ b/common/image-android.c
@@ -17,6 +17,7 @@
 #include <fb_fsl.h>
 #include <asm/setup.h>
 #include <dm.h>
+#include <mmc.h>
 
 #define ANDROID_IMAGE_DEFAULT_KERNEL_ADDR	0x10008000
 
-- 
2.7.4

